Source/Project64/User Interface/Notification Class.h

```diff
@@ -35,6 +35,8 @@ class CNotification :
 	//User Feedback
 	virtual void DisplayMessage(int DisplayTime, const wchar_t * Message) const;
 	virtual void DisplayMessage(int DisplayTime, LanguageStringID StringID) const;
+	virtual void DisplayMessageWide(int DisplayTime, const wchar_t * Message, ...) const;

```

Comment:
I has to removed the ... version before for an upcoming change so I can not add it back now